packages/create-subscription/README.md

```diff
@@ -0,0 +1,186 @@
+# create-subscription
+
+[Async-safe subscriptions are hard to get right.](https://gist.github.com/bvaughn/d569177d70b50b58bff69c3c4a5353f3)
+
+`create-subscription` provides an simple, async-safe interface to manage a subscription.
```

Comment:
This one line summary is too appealing. It doesn't suggest any reasons for why I *shouldn't* use subscriptions.

Reply:
Gotcha. I tried to clarify who _shouldn't_ use it below (in the "**Who should use this?**" section). Any specific wording suggestions?